### PR TITLE
Static ip device in fot

### PR DESCRIPTION
**Pull request by henry54809**

### CODE REVIEW

**Review by didovesei — APPROVED** [review]

File: etc/start_faux [hunk 1]
```diff
@@ -40,6 +40,7 @@ else
 
     ip route add $IPADDR/16 via $GATEWAY dev $IFACE
     ip route add default via $GATEWAY
+    ping $GATEWAY &
```
Comment (grafnu) [inline etc/start_faux:43]: I'm not sure this is a good idea since it will horribly pollute the logs, no?

Comment (henry54809) [reply etc/start_faux]: Changed this so that static ip device only ping during fot tests.

**Comment by henry54809** [comment]
@grafnu tests passed. PTAL

File: etc/Dockerfile.controller [hunk 2]
```diff
@@ -36,6 +36,7 @@ RUN bin/install_forch && rm -rf forch/.git
 COPY daq/ daq/
 RUN daq/bin/setup_dev simple && rm -rf daq/.git
 
+COPY controller/daq_site_config /site
```
Comment (grafnu) [inline etc/Dockerfile.controller:39]: blank line after

File: controller/daq_site_config/mac_addrs/9a02571e8f03/device_config.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (grafnu) [inline controller/daq_site_config/mac_addrs/9a02571e8f03/device_config.json:1]: I would just call this controller/site_config (no daq_ prefix) -- the contents here are really about what goes on the controller machine, and it potentially has more uses outside of DAQ (e.g. used by cloud configuration tools).

**Review by grafnu — APPROVED** [review]

**Pull request merged**